chord_metadata_service/chord/ingest/phenopackets.py

```diff
@@ -262,26 +262,36 @@ def get_or_create_disease(disease) -> pm.Disease:
     return d_obj
 
 
-def get_or_create_diagnosis(diagnosis: dict) -> pm.Diagnosis:
-    # Create GenomicInterpretation
-    genomic_interpretations_data = diagnosis.get("genomic_interpretations", [])
-    genomic_interpretations = [
-        get_or_create_genomic_interpretation(gen_interp)
-        for gen_interp
-        in genomic_interpretations_data
-    ]
-    # disease = pm.Disease.objects.get_or_create(diagnosis["disease"])
-    diag_obj, _ = pm.Diagnosis.objects.get_or_create(
+def get_or_create_interpretation_diagnosis(interpretation: dict) -> pm.Diagnosis:
+    diagnosis = interpretation.get("diagnosis")
+
+    if not diagnosis:
+        return
```

Comment:
this should use a logger to log a warning/error and in the future generate an ingestion warning in the report, i think

Reply:
also this violates the return type hint, since the type hint says there will always be a returned Diagnosis

Reply:
An Interpretation may have no Diagnosis, so this shouldn't be logged as an error. The function creates a Diagnosis from an Interpretation if it contains one, otherwise it returns None. I corrected the return type.